galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/build.gradle

```diff
@@ -7,13 +7,19 @@ description = 'Galasa Selenium Manager'
 dependencies {
     implementation 'dev.galasa:dev.galasa:0.15.0-SNAPSHOT'
     implementation 'dev.galasa:dev.galasa.framework:0.15.0-SNAPSHOT'
+    implementation project(':galasa-managers-comms-parent:dev.galasa.http.manager')
+	implementation project(':galasa-managers-cloud-parent:dev.galasa.docker.manager')
+	implementation project(':galasa-managers-cloud-parent:dev.galasa.kubernetes.manager')
+	implementation project(':galasa-managers-core-parent:dev.galasa.artifact.manager')
```

Comment:
I dont see changes to bnd.bnd to import the packages.   Gradle BND is not like Maven as not as automatic